### PR TITLE
C5 user authentication

### PR DESCRIPTION
**Pull request by jere-mie**
## Description
finished sign up and sign in pages

## Related Issue
- [x] This pull request relates to at least one particular issue

  
  - [Sign Up Page](https://github.com/ps-toronto-team-4/.github/issues/6)
  - [Sign In Page](https://github.com/ps-toronto-team-4/.github/issues/7)

## Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information
None.

### CODE REVIEW

File: constants/Styles.ts [hunk 1]
```diff
@@ -6,11 +6,23 @@ const styles = StyleSheet.create( {
...
         justifyContent: 'center',
-        },
+    },
     title: {
-        fontSize: 20,
+        fontSize: 32,
+        // Not a system font on Android
+        // fontFamily: 'Gill Sans MT',
+        marginHorizontal: 25,
+        marginVertical: '5%',
```
Comment (HKanwal) [inline constants/Styles.ts:15]: I do not think it is good practice to use relative margin.

**Review by HKanwal — CHANGES_REQUESTED** [review]
See requested changes.

File: constants/Styles.ts (hunk 1 above)
Comment (1mozolacal) [reply constants/Styles.ts:15]: Why, I think this is fine as long as it is responsive to different screen sizes

Comment (HKanwal) [reply constants/Styles.ts:15]: Okay, we'll keep it then.

**Comment by 1mozolacal** [comment]
image

Experiencing an issue with the password "qwertyuiop123!A"

**Review by NasihNazeem — CHANGES_REQUESTED** [review]
When inputting an incorrect password, the user gets appropriate feedback mentioning that this is the wrong password, however, this functionality is missing for the incorrect username

**Review by NasihNazeem — APPROVED** [review]

**Pull request merged**